exercises/strings/longestWord/longestWord.js

```diff
@@ -7,15 +7,22 @@
  * @returns {string} The longest word in the input string
  */
 
-function longestWord(string) {
-  // This is your job. :)
+function findLongestWord(str) {
+  var words = str.split(' ');
+  var len = 0;
+
+ words.forEach(function(word){
+   if(word.length > len) {
+     len = word.length;
+   }
+ });
+
+  return len;
 }
 
-if (require.main === module) {
+if (require.main) {
```

Comment:
You don't want to change this line. It won't break things now when you run it from the command line, but it will break things later once we learn about `require(...)`.